src/UI/Art/SpriteArtist/PlayerSetupArtist.java

```diff
@@ -264,7 +265,7 @@ public BufferedImage update(PlayerSetupInfo info)
       if( factionChanged || colorChanged )
       {
         UnitSpriteSet inf = SpriteLibrary.getMapUnitSpriteSet(myControl.getIconicUnit(), info.getCurrentFaction(), info.getCurrentColor());
-        BufferedImage infSprite = inf.sprites[inf.ACTION_IDLE].getFrame(0);
+        BufferedImage infSprite = inf.sprites[AnimState.IDLE.ordinal()].getFrame(0);
```

Comment:
This is the part that will cause conflicts with the `tech` branch. Incidentally, why change this to use `ordinal()`?

Reply:
Because it was type `int` before, now it's type `Enum`

Reply:
Ah. Don't care much either way, but why change it?

Reply:
Because an int variable doesn't have a useful `.ToString()`, which I kind of depend on to read in the movement sprites.